packages/shared/friends/sagas.ts

```diff
@@ -1330,11 +1332,14 @@ function toSocialData(socialIds: string[]) {
 }
 
 function logAndTrackError(message: string, e: any) {
-  logger.error(message, e)
+  const url = getSynapseUrl(store.getState())
+  const variant = new URL(url).host.startsWith('synapse.') ? `Synapse` : `Social Service`
+  const msg = `Service: ${variant} - ${message}`
```

Comment:
It should say it's social service related
```suggestion
  const msg = `Social: ${variant} - ${message}`
```